docs/data-management/qaqc-criteria-unprocessed.md

```diff
@@ -150,17 +157,12 @@ For accurate estimation of task activation, it is essential to check the quality
 
 - [ ] Re-examine the individual visual report of sessions for which `fd_perc` > 50% and double-check that the data does not fall into one of the [exclusion criteria](#functional-mri).
 
+- [ ] Verify that smoothness estimate `FWHM` is roughly consistent across all sessions since they were all acquired with the same protocol. If it is not the case, re-examine the individual visual report of sessions with outlying (both low or high) `FWHM`.
```

Comment:
```suggestion
- [ ] Verify that smoothness estimates (FWHM) are roughly consistent across all sessions since they were all acquired with the same protocol.
    If this is not the case, re-examine the individual visual report of sessions with outlying (both low or high) FWHM.
```